functests/service_account_creation_test.go

```diff
@@ -0,0 +1,34 @@
+package functests
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Service Account Creation", func() {
+	Describe("Service Account", func() {
```

Comment:
I think you can remove this Describe() function.

Reply:
Done